Engine/Entities/Font.hpp

```diff
@@ -33,18 +33,23 @@ const uint32_t EmDash{u'—'};
 const uint32_t HyphenMinus{u'-'};
 const uint32_t NewLine{u'\n'};
 const uint32_t NormalQuote{u'"'};
-const uint32_t CnBegin{u'\x4E00'};
-const uint32_t CnEnd{u'\x9FD5'};
-const uint32_t JpBegin{u'\x3040'};
-const uint32_t JpEnd{u'\x31FF'};
-const uint32_t NumBegin{u'\x0030'};
-const uint32_t NumEnd{u'\x0039'};
-//!,.:;?}·―’”′″∶、。〉》」』！％＇），．：；？｝～
-static const uint32_t NotLineBegin[]{u'\x0021', u'\x002C', u'\x002E', u'\x003A', u'\x003B', u'\x003F', u'\x007D', u'\x00B7', u'\x2015', u'\x2019', u'\x201D', u'\x2032', u'\x2033', u'\x2236', u'\x3001', u'\x3002', u'\x3009', u'\x300B', u'\x300D', u'\x300F', u'\xFF01', u'\xFF05', u'\xFF07',
-                                     u'\xFF09', u'\xFF0C', u'\xFF0E', u'\xFF1A', u'\xFF1B', u'\xFF1F', u'\xFF5D', u'\xFF5E'};
-//([{·‘“《「『＄（．｛
-static const uint32_t NotLineEnd[]{u'\x0028', u'\x005B', u'\x007B', u'\x00B7', u'\x2018', u'\x201C', u'\x300A', u'\x300C', u'\x300E',
-                                   u'\xFF04', u'\xFF08', u'\xFF0E', u'\xFF5B'};
+inline bool isEnLetter(uint32_t ch) {
+	return (ch >= u'a' && ch <= 'z') || (ch >= u'A' && ch <= u'Z');
+}
+inline bool isNumber(uint32_t ch) {
+	return (ch >= '0' && ch <= u'9');
+}
+inline bool isNumberOrLetter(uint32_t ch) {
```

Comment:
Please rename to isNumberOrEnLetter. The rest looks fine to me, I will merge it around the weekend if no issues are found.